content/2022-01-19-brigade-in-5-minutes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Introducing Brigade in 5 Minutes"
+slug: "2022-01-19-brigade-in-5-minutes"
+description: "We've launched a new video training series -- Brigade in 5 Minutes"
+date: "2022-01-19"
+author: "Kent Rancourt"
+summary: "We've launched a new video training series -- Brigade in 5 Minutes."
+---
+
+Good day, Brigadiers!
+
+Today, we just want to make sure everyone knows about the new "Brigade in 5 Minutes" series we have launched on our YouTube channel at [youtube.brigade.sh](https://youtube.brigade.sh) -- that's five minutes plus or minus a couple minutes. 😆
+
+So why this format? Well... we're busy... you're busy... and shorter content is quicker both to produce and to consume, so all of us can get on with our day!
+
+Be sure to check out our [inaugural episode](https://www.youtube.com/watch?v=VFyvYOjm6zc), which is a video adaptation of our [QuickStart documentation](https://quickstart.brigade.sh) and don't forget to like and subscribe!
```

Comment:
Do you want to add a Hugo shortcode for the YouTube video here, so it's embeded in the page?
https://gohugo.io/content-management/shortcodes/#youtube

Reply:
Great idea! I hadn't known about shortcodes. This is awesome!

Reply:
Addressed in ceb959c82a96d15e709c0abf1b57acde376603bd